publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java

```diff
@@ -122,15 +115,15 @@ private InputStream createPublicationRequest(CreatePublicationRequest request) t
             HEADERS, getRequestHeaders(),
             BODY, request
         );
-        return new ByteArrayInputStream(objectMapper.writeValueAsBytes(map));
+        return new ByteArrayInputStream(restApiMapper.writeValueAsBytes(map));
     }
 
     private InputStream emptyCreatePublicationRequest() throws JsonProcessingException {
         Map<String, Object> map = Map.of(
             REQUEST_CONTEXT, createRequestContext(),
             HEADERS, getRequestHeaders()
         );
-        return new ByteArrayInputStream(objectMapper.writeValueAsBytes(map));
+        return new ByteArrayInputStream(restApiMapper.writeValueAsBytes(map));
```

Comment:
these requests can be replaced with the HandlerRequestBuilder but the builder accepts as a parameter an ObjectMapper so there is no gain in the scope of this PR